### PR TITLE
fix: [DEL-3836]: Move Cloud-formation task out of 400-rest

### PR DESCRIPTION
**Pull request by jennyjames21**
You can use the following comments to re-trigger PR Checks

- Compile: `trigger compile`
- runAeriformCheck: `trigger AeriformCheck`
- CodeFormat: `trigger codeformat`
- MessageMetadata: `trigger messagecheck`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- runDockerizationCheck: `trigger dockerizationcheck`
- runAuthorCheck: `trigger authorcheck`
- Checkstyle: `trigger checkstyle`
- PMD: `trigger pmd`
- TI-bootstrap: `trigger ti0`
- TI-bootstrap1: `trigger ti1`
- TI-bootstrap2: `trigger ti2`
- TI-bootstrap3: `trigger ti3`
- TI-bootstrap4: `trigger ti4`
- FunctionalTest1: `trigger ft1`
- FunctionalTest2: `trigger ft2`
- CodeBaseHash: `trigger codebasehash`

---
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/32449)

### CODE REVIEW

**Comment by raghuAtWings** [comment]
*[`930-delegate-tasks/src/main/java/software/wings/delegatetasks/cloudformation/cloudformationtaskhandler/CloudFormationCreateStackHandler.java` line 79 at r1](https://reviewable.io/reviews/harness/harness-core/32449#-N0RWQDXA7opZdmmpZ3G:-N0RWQDXA7opZdmmpZ3H:bcgjvre) ([raw file](https://github.com/harness/harness-core/blob/e76ffd22847fcacf765982c44fa82ee79940175b/930-delegate-tasks/src/main/java/software/wings/delegatetasks/cloudformation/cloudformationtaskhandler/CloudFormationCreateStackHandler.java#L79)):*
> ```Java
> public class CloudFormationCreateStackHandler extends CloudFormationCommandTaskHandler {
>   private int remainingTimeoutMs;
>   public enum Type { TEXT, ENCRYPTED_TEXT }
> ```

Why this change?

**Review by raghuAtWings — COMMENTED** [review]
> Reviewed 24 of 24 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/32449)* status: 0 of 1 approvals obtained.

**Comment by jennyjames21** [comment]
*[`930-delegate-tasks/src/main/java/software/wings/delegatetasks/cloudformation/cloudformationtaskhandler/CloudFormationCreateStackHandler.java` line 79 at r1](https://reviewable.io/reviews/harness/harness-core/32449#-N0RWQDXA7opZdmmpZ3G:-N0RXAU81xJEfITjtnEm:bb5lmfx) ([raw file](https://github.com/harness/harness-core/blob/e76ffd22847fcacf765982c44fa82ee79940175b/930-delegate-tasks/src/main/java/software/wings/delegatetasks/cloudformation/cloudformationtaskhandler/CloudFormationCreateStackHandler.java#L79)):*
<details><summary><i>Previously, raghuAtWings (Raghvendra Singh) wrote…</i></summary><blockquote>

Why this change?
</blockquote></details>

ServiceVariable.Type Enum is used in this file.  ServiceiVaraible.java  extends Base class

**Comment by jennyjames21** [comment]
trigger ft1

**Comment by jennyjames21** [comment]
image

**Comment by jennyjames21** [comment]
image

**Review by jennyjames21 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/32449)* status: 63 file(s) to review. 1. unresolved discussions. 0 of 1 approvals obtained. 

___
*[`930-delegate-tasks/src/main/java/software/wings/delegatetasks/cloudformation/cloudformationtaskhandler/CloudFormationCreateStackHandler.java` line 79 at r1](https://reviewable.io/reviews/harness/harness-core/32449#-N0RWQDXA7opZdmmpZ3G:-N1CQZua2FzrIZZ_iGZB:bvnrpcm) ([raw file](https://github.com/harness/harness-core/blob/e76ffd22847fcacf765982c44fa82ee79940175b/930-delegate-tasks/src/main/java/software/wings/delegatetasks/cloudformation/cloudformationtaskhandler/CloudFormationCreateStackHandler.java#L79)):*
<details><summary><i>Previously, jennyjames21 (Jenny James) wrote…</i></summary><blockquote>

ServiceVariable.Type Enum is used in this file.  ServiceiVaraible.java  extends Base class
</blockquote></details>

updated

File: 260-delegate/src/test/resources/kryo-registrations.txt [hunk 1]
```diff
@@ -983,6 +982,7 @@
 7458:io.harness.beans.terraform.TerraformPlanParam
 7459:software.wings.beans.artifact.ArtifactInput
 7460:software.wings.beans.GitConfig$UrlType
+7462:software.wings.beans.ServiceVariableType
```
Comment (johannesHarness) [inline 260-delegate/src/test/resources/kryo-registrations.txt:985]: as discussed, check with @raghuAtWings whether we should keep the id the same.

**Comment by raghuAtWings** [comment]
*[`260-delegate/src/test/resources/kryo-registrations.txt` line 985 at r8](https://reviewable.io/reviews/harness/harness-core/32449#-N1BzGaj0KO3Hm4jXSZz-r8-985:-N1KwBG43qAGtUZ8PAhl:b8sz5lg) ([raw file](https://github.com/harness/harness-core/blob/39cd3da93b6377c11c17ff2c58f7ca96b5ae4a03/260-delegate/src/test/resources/kryo-registrations.txt#L985)):*
<details><summary><i>Previously, johannesHarness (Johannes Batzill) wrote…</i></summary><blockquote>

as discussed, check with @raghuAtWings whether we should keep the id the same.
</blockquote></details>

same id should be okay

**Review by raghuAtWings — COMMENTED** [review]
> Reviewed 6 of 10 files at r2, 2 of 59 files at r4.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/32449)* status: 65 file(s) to review. 1. unresolved discussions. 0 of 1 approvals obtained.

**Review by raghuAtWings — APPROVED** [review]
> Reviewed 1 of 5 files at r5, 62 of 64 files at r8, 2 of 2 files at r9, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/32449)* status: 1. unresolved discussions.

**Comment by jennyjames21** [comment]
*[`260-delegate/src/test/resources/kryo-registrations.txt` line 985 at r8](https://reviewable.io/reviews/harness/harness-core/32449#-N1BzGaj0KO3Hm4jXSZz-r8-985:-N1L52eS373KaGPKRzWd:b-896fix) ([raw file](https://github.com/harness/harness-core/blob/39cd3da93b6377c11c17ff2c58f7ca96b5ae4a03/260-delegate/src/test/resources/kryo-registrations.txt#L985)):*
<details><summary><i>Previously, raghuAtWings (Raghvendra Singh) wrote…</i></summary><blockquote>

same id should be okay
</blockquote></details>

Done.

**Review by jennyjames21 — COMMENTED** [review]
> Reviewed 13 of 24 files at r1, 1 of 5 files at r5, 63 of 64 files at r8, 2 of 2 files at r9, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/32449)* status: 1. unresolved discussions.

**Review by johannesHarness — APPROVED** [review]
> Reviewed 13 of 24 files at r1, 1 of 5 files at r5, 62 of 64 files at r8, 2 of 2 files at r9, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/32449)* status: :shipit: complete! ✓

**Pull request merged**